docs/dev/backport.md

```diff
@@ -36,3 +36,53 @@ keep the label
 
 5.6. Ask someone else (from QA, or the patch author, or anyone relevant)
 for assistance in selecting among the options
+
+## How to backport a patch
+
+Here are the instructions on how to backport some changes to an older version of Scylla.  
+It's the easiest to show it by an example, so let's look at PR [#12031](https://github.com/scylladb/scylladb/pull/12031), which fixed issue [#12014](https://github.com/scylladb/scylladb/issues/12014).
+
+### Wait for the original change to get merged
+First the original PR should get merged.
+In the example the author wanted to merge the branch `cvybhu:filter_multi_bug` into `scylladb:master`.
+Each PR is submitted on its own branch, which the PR author wants to merge into the `master` branch.
+Before the changes end up on `master`, they are first added to the `next` branch to run some additional tests. Later the `next` branch becomes `master` and the PR is officialy merged.
+
+### A backport PR
```

Comment:
The normal flow is the following:
* PR gets merged to next
* promotion merges next to master
* at this point, if the PR fixes an issue, said issue gets the "Backport Candidate" label
* at some later point a maintainer (Avi) goes through all issues that have said label and backport it to all affected releases, if they see it as a good backport candidate
* the "Backport Candidate" label is removed from the issue

A separate backport PR is required only if the backport is non-trivial: cherry-picking it onto the target branch generates non-trivial merge conflicts. In this case the original author of said PR is asked to prepare a backport PR.

Reply:
Added a note that these instructions are for contributors, who were asked by maintainers to prepare a backport PR.